optuna_dashboard/ts/components/GraphHyperparameterImportances.tsx

```diff
@@ -3,14 +3,65 @@ import React, { FC, useEffect } from "react"
 import { Typography, useTheme, Box, Card, CardContent } from "@mui/material"
 
 import { useParamImportance } from "../hooks/useParamImportance"
-import { useStudyDirections, usePlotlyColorTheme } from "../state"
+import {
+  useStudyDirections,
+  usePlotlyColorTheme,
+  useBackendRender,
+} from "../state"
+import { getPlotAPI, PlotType } from "../apiClient"
 
 const plotDomId = "graph-hyperparameter-importances"
 
 export const GraphHyperparameterImportance: FC<{
   studyId: number
   study: StudyDetail | null
   graphHeight: string
+}> = ({ studyId, study = null, graphHeight }) => {
+  if (useBackendRender()) {
+    return (
+      <GraphHyperparameterImportanceBackend
+        studyId={studyId}
+        study={study}
+        graphHeight={graphHeight}
+      />
+    )
+  } else {
+    return (
+      <GraphHyperparameterImportanceFrontend
+        studyId={studyId}
+        study={study}
+        graphHeight={graphHeight}
+      />
+    )
+  }
+}
+
+const GraphHyperparameterImportanceBackend: FC<{
+  studyId: number
+  study: StudyDetail | null
+  graphHeight: string
+}> = ({ studyId, study = null, graphHeight }) => {
+  const numCompletedTrials =
+    study?.trials.filter((t) => t.state === "Complete").length || 0
+  useEffect(() => {
+    if (studyId === undefined) {
+      return
+    }
+    getPlotAPI(studyId, PlotType.ParamImportances)
+      .then(({ data, layout }) => {
+        plotly.react(plotDomId, data, layout)
+      })
+      .catch((err) => {
+        console.error(err)
+      })
```

Comment:
To avoid calling the API directly in `useEffect`, a new hook `usePlot` has been introduced (`optuna_dashboard/ts/hooks/usePlot.ts`).
Could you change the implementation to use this? (FYI, see the implementation of `optuna_dashboard/ts/components/GraphContour.tsx`)

Reply:
Thanks, I applied this change: https://github.com/optuna/optuna-dashboard/pull/805/commits/4ceeb1bc7a056eebabd458860b914b5b40b31d8a